uint8 FrameHead
float64 Pitch
float64 Yaw
uint8 FlagFound
uint8 FlagFire
uint8 FrameTail